server/src/router.ts

```diff
@@ -142,7 +143,16 @@ export class Router {
       }),
       tokenRouter
     );
-
+    app.use(
+      "/search",
```

Comment:
good use of express router, very minor request to change search to something a little more descriptive like image-search or google-image-search